### DISCUSSION TITLE
Create Similar Label As That of Material UI

### DISCUSSION THREAD

**Discussion by saidhasyim** [react-hook-form/react-hook-form#8986] [Q&A] [answered]
Hello, i started using React Hook Form Input. 

I am referring to the sample code in https://codesandbox.io/s/react-hook-form-hookforminput-rzu9s?file=/src/index.js

I'm trying to make the input label of a dropdown Select similar to that in Material UI. in the below screenshot.
![image](https://user-images.githubusercontent.com/111839161/188561191-a5ac9428-658a-475e-b9ba-90a5da4812a3.png)

Would you advise me how to achieve that? to keep the input field consistent with the rest of my fields.

Thanks very much

**Comment by Moshyfawn** [comment, marked as answer]
RHF is not a UI library. It doesn't manage what your input looks like. With MUI, you want to go to their docs and see their implementation.

One tip I can give you: try using the `TextField` component; pass a boolean prop `select` and `label` with your desired text.

You can learn more following [this link](https://mui.com/material-ui/react-text-field/#select)

**Reply by saidhasyim** [reply]
it works!! thanks so much
